### PR TITLE
Update guide of demo server

### PR DESCRIPTION
**Pull request by ywk253100**

### CODE REVIEW

File: docs/demo_server.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+ - We will **CLEAN AND RESET** the server every **TWO Days**.
+ - You can only experience the nonadmin functionalities on this server. Please follow the **[Installation Guide](installation_guide.md)** to setup a Harbor server to try more advanced features.
+ - Please do not push large images(>100MB) as the server has limited storage.
+
+If you encounter any problems during using the demo server, please contact us at <img alt="email" src="img/harbor_email.png" valigin="middle" height="18"/>.
+
+**Usage**
+
+ - 1> The address of the demo server is [https://demo.goharbor.io](https://demo.goharbor.io)
+ - 2> You can registry a new user by yourself
```
Comment (hainingzhang) [inline docs/demo_server.md]: registry -> register

File: docs/demo_server.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
+# Demo Server Guide
+
+**Important!** 
+ - Please note that this demo server is **ONLY** for your trial of Harbor functionalities. 
+ - Please **DO NOT** upload any sensitive images to this server. 
+ - We will **CLEAN AND RESET** the server every **TWO Days**.
+ - You can only experience the nonadmin functionalities on this server. Please follow the **[Installation Guide](installation_guide.md)** to setup a Harbor server to try more advanced features.
```
Comment (hainingzhang) [inline docs/demo_server.md]: non-admin

File: README.md [hunk 3]
```diff
@@ -38,6 +38,8 @@ Refer to **[User Guide](docs/user_guide.md)** for more details on how to use Har
 **Developer Group:** Join Harbor developer group: [harbor-dev@googlegroups.com](https://groups.google.com/forum/#!forum/harbor-dev) for discussion on Harbor development and contribution. To subscribe, send an email to [harbor-dev+subscribe@googlegroups.com](mailto:harbor-dev+subscribe@googlegroups.com).  
 **Slack:** Join Harbor's community for discussion and ask questions: [VMware {code}](https://code.vmware.com/join/), channel: [#harbor](https://vmwarecode.slack.com/messages/harbor).
 
+**Demo Server:** Harbor provides a demo server with latest Harbor stable build installed. If you want to have a try, please refer to **[Demo Server](docs/demo_server.md)** for more details.
```
Comment (hainingzhang) [inline README.md]: Harbor provides a demo server with latest Harbor 
-> Harbor provides a demo environment with the latest Harbor

**Comment by ywk253100** [comment]
@hainingzhang Updated.

**Pull request merged**